### ISSUE TITLE
Enhancement: Soundcloud is outdated.

### ISSUE THREAD

**Issue by ExcodeFNF** [FunkinCrew/Funkin#3467]
### Issue Checklist

- [X] I have properly named the enhancement
- [X] I have checked the issues/discussions pages to see if the enhancement has been previously suggested

### What is your suggestion, and why should it be implemented?

![image](https://github.com/user-attachments/assets/eda166cc-9857-4337-b9ba-caafe74d82d4)
As seen on this photo, Darnell and Lit Up have not yet been placed into the playlist, either by error or accidentally forgotten. Please resolve as soon as possible!

**Comment by JVNpixels** [comment]
I agree, they also need to upload the Expansion 2 Instrumentals for Spotify and other streaming services.

**Comment by KingMB-XJ** [comment]
Pico (Pico Mix) is also missing from all versions of the Vol 4. OST, including Soundcloud.

**Comment by ExtraCode75** [comment]
Darnell has been uploaded to the Soundcloud and Pico (Pico Mix) has been put in Expansion Vol. 2 of the OST.

**Comment by ExtraCode75** [comment]
Lit Up is still missing from the OST.
